.circleci/config.yml

```diff
@@ -34,6 +38,17 @@ workflows:
                 # Long living branches
                 - main
                 # 👇Add your branch here if benchmarking matters to your work
+
+  coverage:
+    jobs:
+      - coverage:
+          filters:
+            branches:
+              only:
+                # Long living branches
+                - main
+                # 👇Add your branch here if code coverage matters to your work
```

Comment:
Let's stick to plain old ASCII, I already removed those emojis in some different PR. :)
And I agree with Simon, I'd remove filter completely. Or maybe even put `except` clause for `main` and `tags` - since coverage here is for devs, and not to fullfill some corpo requirements.

Reply:
I'll add new emojis elsewhere, no worries

Reply:
Thanks, forgot about this change. I'll remove the filter.